py/desitarget/mock/io.py

```diff
@@ -321,17 +321,15 @@ def read_wd100pc(mock_dir, target_type, mock_name=None):
     desitarget.io.check_fitsio_version()
     C_LIGHT = 299792.458
 
-    mock_name = 'mock_wd100pc.fits'
+    mock_name = 'mock_100pc.fits'
```

Comment:
These currently override the `mock_name` input to the function; I think you want
```python
if mock_name is None:
    mock_name = 'mock_100pc.fits'
```